.env.sample

```diff
@@ -1,4 +1,4 @@
-FX_REMOTE_SETTINGS_WRITER_SERVER="https://settings-writer.prod.mozaws.net/v1"
+SERVER="https://remote-settings-dev.allizom.org/v1"
 FX_REMOTE_SETTINGS_WRITER_USER=""
 FX_REMOTE_SETTINGS_WRITER_PASS=""
```

Comment:
Now that we are at it...

Looking at our specification: 
https://remote-settings.readthedocs.io/en/latest/support.html#how-do-i-automate-the-publication-of-records-forever

These two should be passed as `AUTHORIZATION=username:password`

Reply:
yeah, I know .. just a little more work for me to refine this than just "making the script work" on GKE. I'll see if I can get this fixed some time in the next days.